package.json

```diff
@@ -34,8 +34,8 @@
     "postcss-import": "^8.1.0",
     "postcss-loader": "^0.9.1",
     "precss": "^1.4.0",
-    "react": "^15.1.0",
-    "react-dom": "^15.1.0",
+    "react": "15.1.0",
```

Comment:
This is an unrelated change I accidentally snuck in. I'm totally fine with not pinning the last version bit.